main.ledger

```diff
@@ -1545,12 +1580,26 @@
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: a2875e2a6987e90e4817b121274cfdf6.pdf
      
+2016/02/03 Someone didn't cash their check
+    Assets:Bank:Checking                              $1900.00
+    Expenses:Salary
+    ; Receipt: 9b1b2f9c1dcc2d6a19757f88fa422fcb.png
```

Comment:
You sure it was for $1900 and not $1920? I'm not seeing anything for $1900
in my email after an initial skim.

On Tue, Apr 12, 2016 at 8:40 AM, JESSICA <notifications@github.com> wrote:

> In main.ledger
> <https://github.com/hackclub/finances/pull/12#discussion_r59399024>:
>
> > @@ -1545,12 +1580,26 @@
> >      Liabilities:Reimbursement:Zach Latta
> >      ; Receipt: a2875e2a6987e90e4817b121274cfdf6.pdf
> >
> > +2016/02/03 Someone didn't cash their check
> > +    Assets:Bank:Checking                              $1900.00
> > +    Expenses:Salary
> > +    ; Receipt: 9b1b2f9c1dcc2d6a19757f88fa422fcb.png
>
> you should have gotten some sort of email from wells fargo; we should use
> that as the receipt instead
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/finances/pull/12/files/050535646cac2b1339431fbe861ff24075f4fdb7#r59399024>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>